server/api/controllers/user.js

```diff
@@ -407,7 +409,7 @@ const requestResetPassword = (req, res) => {
 			let errorMessage = errorMessageConverter(err);
 
 			if (errorMessage === USER_NOT_FOUND) {
-				errorMessage = `Password request sent to: ${email}`;
+				errorMessage = `Email ${email} not found`;
```

Comment:
@hoan-bitholla these messages are done this way so the user sending the request cant tell whether or not an email exists in the system. The returned message should be the same regardless.

Reply:
OK, sure! I'll revert it